### ISSUE TITLE
Why are listings arrays of listings?

### ISSUE THREAD

**Issue by JustinDrake** [OpenBazaar/openbazaar-go#376]
See for example this listings http://gateway.ob1.io/ipns/QmSLTwZQPxX1peJVXxBBKTrVGVMKSSyom5sTDHxtVricvs/listings/ski-set.json.

It's an array of listings:

```
{
    "vendorListings": [
        {
            "slug": "ski-set",
            "vendorID": {
                "guid": "QmSLTwZQPxX1peJVXxBBKTrVGVMKSSyom5sTDHxtVricvs",
                "blockchainID": "skishop",
                "pubkeys": {
                    "guid": "CAESIMHAzLNHF4P8vJY13fEFG/IZoSugC2HKh/8ml9o1EgIi",
                    "bitcoin": "A4al6HwsV8bJmlFTNf3Pb63dl47S3Y6sAF31BaSP8RkB"
                },
                "bitcoinSig": "MEUCIQCsBVcebEzrsoUs8fGO03/YvN5Lq+Awc7N2lZBGk6hObgIgTOnvny4OmjtJc3TfhfOp5EYypqKN18v4ZYZTtNtzx+I="
            },
```

Why are listings arrays of listings?

**Comment by hoffmabc** [comment]
The field is vendorListings...am I missing something here? It's an array of one listing? Is that the issue?

**Comment by JustinDrake** [comment]
Yeah, what's this `vendorListings` field all about? Can there more than one listing?

**Comment by hoffmabc** [comment]
@cpacia you'll have to answer that one. is this intended or just a copy-pasta from the listings response?

**Comment by cpacia** [comment]
I thought there was an open issue for this already.

**Comment by cpacia** [comment]
closed with #507
